CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 
 ### Bug Fixes:
 - Fixed a bug in SQL simplifier that didn’t handle Column alias and join well in some cases. See https://github.com/snowflakedb/snowpark-python/issues/658 for details.
+- Fixed a bug in SQL simplifier that generated wrong column names for function calls, NaN and INF.
```

Comment:
@sfc-gh-mabrennan We happened to find a bug while 1.1.0 hasn't been released. Could you review it ASAP?